frontend-cert/js-projects/rock-paper-scissor/script.js

```diff
@@ -0,0 +1,52 @@
+let userChoice = "rock";
```

Comment:
```suggestion
let userChoice = prompt("Choose between rock, paper, and scissors: ");
```
We could take the user choice as input. Then, in case the choice is not valid, a default case can be added to the switch.
What do you think?